### PR TITLE
[Fluent] Add editor styles

### PR DESCRIPTION
**Pull request by epetrow**
Related to: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/editor/_theme.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+        .k-row-resize-handle-wrapper {
+            .k-row-resize-marker {
+                background-color: #00b0ff;
+            }
+        }
+
+        // Hover & Actve state
+        &:hover,
+        &.k-state-active,
```
Comment (TeyaVes) [inline packages/fluent/scss/editor/_theme.scss:54]: remove -state classes

**Comment by TeyaVes** [comment]
@epetrow , Looks, great, just a few minor fixes needed. Also can you please double check the font-family and font-size dropdowns in editor-jquery tests? It seems that the buttons are missing border-top.

**Review by TeyaVes — CHANGES_REQUESTED** [review]

**Comment by epetrow** [comment]
> @epetrow , Looks, great, just a few minor fixes needed. Also can you please double check the font-family and font-size dropdowns in editor-jquery tests? It seems that the buttons are missing border-top.

Seems like this is only the case with the visual previews here, if you checkout the branch and run the tests locally the border is there and the table wizard looks good. I've tested it with both Chrome and Firefox.

**Review by TeyaVes — APPROVED** [review]

File: packages/fluent/scss/editor/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+$kendo-editor-text: var( --kendo-component-text, initial ) !default;
+/// The background color of the editor
+/// @group editor
+$kendo-editor-bg: var( --kendo-component-bg, initial ) !default;
+/// The color of the border around editor
+/// @group editor
+$kendo-editor-border: var( --kendo-component-border, initial ) !default;
+/// The placeholder text color of the editor
+/// @group editor
+$kendo-editor-placeholder-text: $kendo-input-placeholder-text !default;
```
Comment (TeyaVes) [inline packages/fluent/scss/editor/_variables.scss:27]: use css var here and bellow

**Review by TeyaVes — CHANGES_REQUESTED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**